lib/ansible/plugins/action/__init__.py

```diff
@@ -1045,6 +1045,10 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
             display.debug("_low_level_execute_command(): changing cwd to %s for this command" % chdir)
             cmd = self._connection._shell.append_command('cd %s' % chdir, cmd)
 
+        # https://github.com/ansible/ansible/issues/68054
+        if executable:
```

Comment:
https://github.com/ansible/ansible/blob/eaf648e60093505d47b2918e58453fe9181ea517/lib/ansible/plugins/action/__init__.py#L1063-L1070
The above code makes me wonder if this should too only be done `if self._connection.allow_executable`?

Reply:
After looking further through the code, the same seems to be already handled in `_build_success_command()` by `noexe` arg.